gtk4/src/subclass/widget.rs

```diff
@@ -123,3 +126,127 @@ unsafe extern "C" fn widget_direction_changed<T: ObjectSubclass>(
 
     imp.direction_changed(&wrap, dirwrap)
 }
+
+pub trait WidgetSubclass
+where
+    Self: ObjectSubclass + WidgetImpl,
+{
+    unsafe fn set_template_bytes(
```

Comment:
API question: should these functions be `unsafe` for the developers, or should they be "safe" and use `unsafe` internally like other bindings?

Reply:
The only place where you can safely get a mutable reference to the class struct is in `class_init()`. Is it safe to use there in any case?